api/graphql/mutations.js

```diff
@@ -274,3 +274,11 @@ export function flagInappropriateContent (userId, {
   }).save()
   .then(() => ({success: true}))
 }
+
+export function createCommunity (userId, data) {
+  data.slug = data.name
```

Comment:
Since the front-end asks for the slug in its own step, we shouldn't need to do anything special here. (The front-end should be doing something like what we do on the current site: https://github.com/Hylozoic/hylo-redux/blob/master/src/containers/CreateCommunity.js#L48)

We should validate, though, that the slug doesn't already exist; otherwise the creation will fail with a database error.